lib/internal/modules/esm/loader.js

```diff
@@ -18,7 +18,10 @@ const { validateString } = require('internal/validators');
 const ModuleMap = require('internal/modules/esm/module_map');
 const ModuleJob = require('internal/modules/esm/module_job');
 
-const { defaultResolve } = require('internal/modules/esm/resolve');
+const {
+  defaultResolve,
+  DEFAULT_CONDITIONS,
```

Comment:
I would like to remove `DEFAULT_CONDITIONS` on `module.exports` and use

```js
function moduleResolve(specifier, base, conditions = ['import','node']) {
  // ...
}
```

Reply:
The issue is that we need to pass those into a userland function, not just into our own default implementation. So I don't think that can be replaced with default arguments. Unless of course I'm missing something about the suggestion. :)